homeassistant/components/homee/cover.py

```diff
@@ -44,7 +44,7 @@ def get_cover_features(
     """Determine the supported cover features of a homee node based on the available attributes."""
     features = CoverEntityFeature(0)
 
-    if open_close_attribute.editable:
+    if (open_close_attribute is not None) and open_close_attribute.editable:
```

Comment:
open_close_attribute can't be `None` since it's typed only to be `HomeeAttribute`